docs/track_types.rst

```diff
@@ -45,7 +54,16 @@ Intervals can visually encode information using the following three ``options``:
 ``colorEncodingRange: array``
     A tuple defining the minimum and maximum range value for color encoding.
 
-Here is an example snippet
+``plusStrandColor``
+    For stranded mode, the color of the plus strand entries
+
+``minusStrandColor``
+    For stranded mode, the color of the minus strand entries
+
+``fillColor``
```

Comment:
Wasn't fill color removed? I am confused haha

Reply:
No, not removed. My mistake above. Sorry about that.